ow_lander/src/ow_lander/actions.py

```diff
@@ -111,27 +111,27 @@ def ground_detect_cb():
         self._set_succeeded("No ground detected", final=Point(), success=False)
 
 
-class UnstowServer(ArmTrajectoryMixin, ActionServerBase):
+class ArmUnstowServer(ArmTrajectoryMixin, ActionServerBase):
 
-  # UNIFICATION TODO: rename "Stow" to "ArmStow"
-  name          = 'Unstow'
-  action_type   = ow_lander.msg.UnstowAction
-  goal_type     = ow_lander.msg.UnstowGoal
-  feedback_type = ow_lander.msg.UnstowFeedback
-  result_type   = ow_lander.msg.UnstowResult
+  # UNIFICATION TODO: rename "ArmStow" to "ArmArmStow"
```

Comment:
If unification turns `Stow` to `ArmStow` it follows that `ArmArmStow` is double-unification 🤣 
```suggestion
```
I kid. This comment is no long needed.

Reply:
Search/replace error.  Both comments removed.